setup.py

```diff
@@ -24,6 +24,10 @@
         'sonic_sfp',
         'sonic_thermal',
     ],
+    install_requires=[
+        'sonic-config-engine',  # For portconfig. TODO: Remove this dependency
```

Comment:
Note: This dependency was always needed, but was never explicitly specified. I added it here along with a TODO to eventually remove it (portconfig should eventually be moved into sonic-py-common).